keyboards/gray_studio/space65/keymaps/billiams/keymap.c

```diff
@@ -0,0 +1,125 @@
+   * |------------------------------------------------------------------------------------------------+
+   * |      |       |       |                                  |      |  Fn2  |  HOME  | PGDN  |  END |
+   * `------------------------------------------------------------------------------------------------'
+   */
+
+[1] = LAYOUT( \
+             KC_TRNS, KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,  KC_TRNS, KC_TRNS, KC_MFFD, \
+             KC_TRNS, RGB_TOG, RGB_MOD, RGB_HUD, RGB_HUI, RGB_SAD, RGB_SAI, RGB_VAD, RGB_VAI, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_MRWD,          \
+             KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_LEFT, KC_DOWN, KC_UP, KC_RIGHT, KC_TRNS, KC_TRNS, KC_MPLY, KC_VOLU,                   \
+             KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_BRID, KC_BRIU, KC_TRNS, KC_TRNS, KC_PGUP, KC_VOLD, \
+             KC_TRNS, KC_TRNS, KC_TRNS,                   KC_TRNS, KC_TRNS, KC_TRNS,          KC_TRNS, MO(2), KC_HOME, KC_PGDN, KC_END \
+    ),
+
+  /* 2nd Layer
+   * ,------------------------------------------------------------------------------------------------.
+   * |     |     |     |     |     |     |     |     |     |     |     |     |     |     |     |      |
+   * |------------------------------------------------------------------------------------------------+
+   * |        |     |     |     |     |     |     |     |     |      |     |     |      |      |      |
+   * |------------------------------------------------------------------------------------------------+
+   * |         |     |     |     |     |     |     |     |    |    |      |      |            |       |
+   * |------------------------------------------------------------------------------------------------+
+   * |           |     |     |     |     |     |     |     |     |     |     |     |     |    | RESET |
+   * |------------------------------------------------------------------------------------------------+
+   * |      |       |       |                                   |      |       |       |       |      |
+   * `------------------------------------------------------------------------------------------------'
+   */
+
+[2] = LAYOUT( \
+             KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, \
+             KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,          \
+             KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,                   \
+             KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,          \
```

Comment:
The `RESET` keycode described in your readme isn't present either.

```suggestion
             KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, RESET,            \
```